hi-ml-azure/run_pytest.py

```diff
@@ -68,6 +68,8 @@ class RunPytestConfig(param.Parameterized):
         default="",
         doc="A folder name that should be added to sys.path. The folder name should be relative to repository root."
     )
+    strictly_aml_v1: bool = param.Boolean(default=False, doc="If True, use AzureML v1 SDK. If False (default), use "
```

Comment:
[Minor] Seems like it makes more sense to have default as True from the rest of the code.

Reply:
I think we should be using the SDK v2 now where possible. Are there specific places where it contradicts this? (If you're judging by the number of tests where I've set `strictly_aml_v1=True` it's because I didn't want to change the behaviour of the tests)

Reply:
Yes, exactly! I have a comment somewhere in the tests that says that I finally got why you want to set it as False by default.